code/modules/uplink/uplink_items.dm

```diff
@@ -1328,6 +1321,27 @@ GLOBAL_LIST_EMPTY(uplink_items) // Global list so we only initialize this once.
 	cost = 20
 	cant_discount = TRUE
 
+/datum/uplink_item/badass/costumes
+	surplus = 0
+	include_modes = list(/datum/game_mode/nuclear)
+	cost = 4
+	cant_discount = TRUE
```

Comment:
I don't think there's anything preventing this particular type from showing up in the purchase list?

Reply:
Entries with no item var won't show up.